tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -202,6 +206,18 @@ final class TestingServer(
     }
   }
 
+  def formatting(filename: String): Future[util.List[TextEdit]] = {
```

Comment:
We should update in-memory buffers to use the new text file contents. That avoids testing against `List[TextEdit]`

Reply:
Ah yes, I thought of it then I forgot completely 😅